openassessment/assessment/models/peer.py

```diff
@@ -248,13 +249,31 @@ def get_waiting_step_details(
                 'graded_by': 2
             }
         """
+        Count, Q = models.Count, models.Q
```

Comment:
What is this doing?

Reply:
I didn't want to have to type models.Count and models.Q every time I used them so I just assigned them to these variables

Reply:
Looking at my final result it's not super necessary. At some point I had a big chain of these filter objects and it was super ugly. I'll remove.